pkg/bmc/idrac_virtualmedia.go

```diff
@@ -90,3 +92,52 @@ func (a *redfishiDracVirtualMediaAccessDetails) VendorInterface() string {
 func (a *redfishiDracVirtualMediaAccessDetails) SupportsSecureBoot() bool {
 	return true
 }
+
+func (a *redfishiDracVirtualMediaAccessDetails) ValidateBootInterface(bmcCreds Credentials) (bool, error) {
+	bmcConfig := gofish.ClientConfig{
+		Endpoint: getRedfishAddress(a.bmcType, a.host),
+		Username: bmcCreds.Username,
+		Password: bmcCreds.Password,
+		Insecure: a.disableCertificateVerification,
+	}
+
+	client, err := gofish.Connect(bmcConfig)
+	if err != nil {
+		return false, err
+	}
+	defer client.Logout()
+
+	service := client.Service
+
+	managers, err := service.Managers()
+	if err != nil {
+		return false, err
+	}
+
+	for _, manager := range managers {
+		vmediaList, err := manager.VirtualMedia()
+		if err != nil {
+			return false, err
+		}
+		if vmediaList == nil {
+			continue
+		}
+
+		for _, vmedia := range vmediaList {
+			for _, media := range vmedia.MediaTypes {
+				if media != "CD" {
```

Comment:
Is there a constant for this maybe?

Reply:
yeah, I will switch to use the constant from  https://github.com/stmcginnis/gofish/blob/main/redfish/virtualmedia.go#L36

Reply:
Done